table/block_based_table_builder.cc

```diff
@@ -104,36 +104,36 @@ bool GoodCompressionRatio(size_t compressed_size, size_t raw_size) {
 
 // format_version is the block format as defined in include/rocksdb/table.h
 Slice CompressBlock(const Slice& raw,
-                    const CompressionOptions& compression_options,
+                    const CompressionContext& compression_ctx,
                     CompressionType* type, uint32_t format_version,
```

Comment:
@miasantreble The problem is that this function needs an output argument in case the compression ratio is either not too good or not supported in which case we return the actual compression used(not used) via type. And this is used at the BlockBasedTableBuiilder